src/l500/l500-private.h

```diff
@@ -353,6 +365,19 @@ namespace librealsense
         {
             rgb_common common;
             resolutions_rgb resolution;
+
+            size_t get_expected_size(const std::vector<uint8_t> & raw_data_vec) const
```

Comment:
I don't like that this function repeats... can we use this code within the calling function?

Reply:
I am not sure I understand the comment.
The calling function is check_calib template.
It expect that the template class will have this function.
So what did you mean?

Reply:
Call me and we'll talk

Reply:
Fixed as agreed, remove function parameter and add a comment